lambda/custom/package.json

```diff
@@ -5,7 +5,17 @@
   "main": "index.js",
   "scripts": {
     "setup": "bash ../../bin/setup.sh",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "clean": "rimraf .build",
+    "compile": "tsc --build tsconfig.json --pretty",
+    "build-final": "cpy package.json .build && cpy yarn.lock .build && cpy .env .build && cd .build/ && yarn install --production=true",
```

Comment:
Now that we can control which files are copied into the bundle, I was able to re-enable ```dotenv```.